zebra-network/src/peer_set/inventory_registry.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{protocol::external::InventoryHash, BoxedStdError};
```

Comment:
There aren't many comments in this file - no module comment, and no function comments.

Do you think they would help us maintain the file in future?

Reply:
Personally, I don't think comments would help much, because the code is already fairly explicit about what it's doing, and the RFC describes the motivation for why it's doing that.

Reply:
_tfw i just finished commenting everything_

Reply:
Can we link to the RFC from a module comment?

Reply:
way ahead of ya https://github.com/ZcashFoundation/zebra/pull/987/files#diff-6b20487f053c3c6092545041cefa82deR3